frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -36,6 +38,40 @@ const WARNING_TYPE_RENDERER = {
             </>
         )
     },
+    cannot_merge_with_illegal_distinct_id: function Render(warning: IngestionWarning): JSX.Element {
+        const details = warning.details as {
+            illegalDistinctId: string
+            otherDistinctId: string
+        }
+        return (
+            <>
+                Refused to merge an illegal distinct_id{' '}
+                <Link to={urls.person(details.illegalDistinctId)}>{details.illegalDistinctId}</Link> with{' '}
+                <Link to={urls.person(details.otherDistinctId)}>{details.otherDistinctId}</Link> via an $identify or
+                $create_alias call
+            </>
+        )
+    },
+    cannot_merge_attepts_exhausted: function Render(warning: IngestionWarning): JSX.Element {
+        const details = warning.details as {
+            sourcePersonDistinctId: string
+            targetPersonDistinctId: string
+            error: string
+        }
+        return (
+            <>
+                Merging <Link to={urls.person(details.sourcePersonDistinctId)}>{details.sourcePersonDistinctId}</Link>{' '}
+                into <Link to={urls.person(details.targetPersonDistinctId)}>{details.targetPersonDistinctId}</Link> via
+                an $identify or $create_alias call failed with error: {details.error}.
+            </>
+        )
+    },
+    skipping_event_invalid_uuid: function Render(warning: IngestionWarning): JSX.Element {
+        const details = warning.details as {
+            eventUuid: string
+        }
+        return <>Refused to process event with invalid uuid: {details.eventUuid}.</>
```

Comment:
Problem: The eventUuid might not be displayed in a useful way here given it might be an empty string, null, a number, etc.

Suggestion: `code block style` this and JSON stringify the value before displaying.